packages/page-accounts/src/modals/Create.tsx

```diff
@@ -381,10 +386,10 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
                     value={derivePath}
                   />
                   {deriveValidation?.error && (
-                    <article className='error'>{deriveValidation.error}</article>
+                    <article className='error'>{errorIndex[deriveValidation.error]}</article>
```

Comment:
The caught error case is not handled, so `errorIndex[...] || deriveValidation.error`

Reply:
Indeed, just realized that. Thanks. Should be fixed by https://github.com/polkadot-js/apps/pull/3663/commits/ef2d6270d30a1a31b81fc5e976b074a41fee9cb0